opensrp-chw/src/main/java/org/smartregister/chw/activity/HivProfileActivity.java

```diff
@@ -559,13 +559,13 @@ protected void startAncRegister() {
     public void startHivRegistrationDetailsActivity() {
         Intent intent = new Intent(this, BaseHivFormsActivity.class);
         intent.putExtra(org.smartregister.chw.hiv.util.Constants.ActivityPayload.BASE_ENTITY_ID, getHivMemberObject().getBaseEntityId());
-
-        String formName;
-        if (getHivMemberObject().getGender().equalsIgnoreCase("male")) {
-            formName = CoreConstants.JSON_FORM.getMaleHivRegistration();
-        } else {
-            formName = CoreConstants.JSON_FORM.getFemaleHivRegistration();
-        }
+        //TODO: fix on merge
+        String formName = null;
+//        if (getHivMemberObject().getGender().equalsIgnoreCase("male")) {
+//            formName = CoreConstants.JSON_FORM.getMaleHivRegistration();
+//        } else {
+//            formName = CoreConstants.JSON_FORM.getFemaleHivRegistration();
+//        }
```

Comment:
Yes, these were there because of the updated version of core but the changes weren't yet merged in the default branch. Seems like it has already been merged so I can fix this